Application/EdFi.Ods.Api/Startup/OdsStartupBase.cs

```diff
@@ -64,6 +64,8 @@ public abstract class OdsStartupBase
     {
         private const string CorsPolicyName = "_development_";
 
+        private static readonly string _identityManagementClaimType = $"{EdFiConventions.EdFiOdsResourceClaimBaseUri}/services/identity";
```

Comment:
Just calling out that the identity claim refers to a service endpoint rather than a data management endpoint. This is separated out using the `/services/` segment in the claim name so that we can significantly reduce the number of ClaimsPrincipal claims that we are creating (since we don't need to use those for data management authorization, only for ASP.NET identities controller security).